q2_composition/assets/run_ancombc.R

```diff
@@ -82,34 +82,28 @@ otu <- otu_table(otu_file, taxa_are_rows = TRUE)
 md <- sample_data(metadata_file)
 row.names(md) <- rownames(metadata_file)
 
-if (reference_levels == "") {
-  reference_levels <- NULL
-}
-
 intercept_groups <- c()
 # split the reference_levels param into each column and associated level order
-if (!is.null(reference_levels)) {
```

Comment:
Same as the python sibling branch - this is no longer needed since `reference_levels` will never be none